### DISCUSSION TITLE
Shining ?

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#133] [Q&A] [answered]
Hello, 
I checked other discuss and I don't found anything about this : is that possible to make an object shining ? Like a lightning ? Fire ?

**Comment by pragma37** [comment, marked as answer]
Hi!
If you mean an emission-like shader, just plug whatever color you want into the PixelOutput Color.
If you mean glow, this file has an example:
https://github.com/bnpr/Malt/discussions/94#discussioncomment-933014

**Reply by Airioo** [reply]
I mean Glow, like bloom effect in blender !
Thanks will check it out this night !

**Reply by Airioo** [reply]
And is that possible to convert a node to code ? Cauz it's easier to duplicate and change (for colors for example)

**Reply by Airioo** [reply]
Oh okay I see : while you check the box in material panel you can change individual color ! Thanks ! Your amazing !!!

**Reply by aVersionOfReality** [reply]
> And is that possible to convert a node to code ? Cauz it's easier to duplicate and change (for colors for example)

Yes, see this thread: https://github.com/bnpr/Malt/discussions/129

**Reply by Airioo** [reply]
Thanks !
